README.md

```diff
@@ -1,33 +1,50 @@
 # NPI_FILES
+
 repo to automate NPI file download, extract and load into a database and create an api for endpoint
 
-## Podman setup in script  
+## Podman setup in script
 
 work in progress... `pod_setup.sh`
 
-## Local Development for Django  
+## Local Development for Django
+
+Need to set up Postgres, either locally or in a container. The container is my choice as it makes it easier when testing with the containerized django app. The instructions below are specific to local Postgres environment.
+
+## Prerequisite
+
+This requirement will have you update the `.env.dummy` file with appropriate values and rename it to `.env` so you don't check in any passwords or settings. These values are referenced in Django's `settings.py` file
```

Comment:
let's change this from renaming to copying otherwise we will run into the same issues when trying to save/check in.